dbt_subprojects/daily_spellbook/models/xrpl/transfers/tokens_xrpl_transfers.sql

```diff
@@ -0,0 +1,289 @@
+{{
+    config(
+        schema = 'tokens_xrpl',
+        alias = 'transfers',
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["xrpl"]\',
+                                    "sector",
+                                    "tokens",
+                                    \'["krishhh"]\') }}'
+    )
+}}
+
+
+WITH all_transfers AS (
+    -- 1. Payment Transfers
+    SELECT 
+        unique_key
+        ,blockchain
+        ,block_month
+        ,block_date
+        ,block_time
+        ,ledger_index
+        ,tx_hash
+        ,tx_index
+        ,evt_index
+        ,transfer_type
+        ,token_standard
+        ,transaction_type
+        ,transaction_result
+        ,sequence
+        ,fee
+        ,tx_from
+        ,tx_to
+        ,from_address
+        ,to_address
+        ,issuer
+        ,currency
+        ,currency_hex
+        ,symbol
+        ,amount_requested_raw
+        ,amount_delivered_raw
+        ,amount_requested
+        ,amount_delivered
+        ,amount
+        ,partial_payment_flag
+        ,price_usd
+        ,amount_usd
+    FROM {{ ref('tokens_xrpl_payments') }}
+    {% if is_incremental() %}
+    WHERE {{ incremental_predicate('block_date') }}
+    {% endif %}
+    
+    UNION ALL
```

Comment:
plz use the for loop approach for this to keep it clean. there are plenty of examples in spellbook, one being the chain-level views in dex trades lineage.